### ISSUE TITLE
Wrong vertical ToolBar style.

### ISSUE THREAD

**Issue by p1eXu5** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2788]
### Bug explanation

In the MaterialDesignToolBar style DockPanel.Dock of the Grid with Popup stay the same when ToolBar orientation became Vertical. This leads to extra right margin of a vertical ToolBar. To fix this problem an additional setter is needed:

```xml
<!-- MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToolBar.xaml -->
...
<Style x:Key="MaterialDesignToolBar" ...>
    ...
    <ControlTemplate.Triggers>
        ...
        <Trigger Property="Orientation" Value="Vertical">
            <Setter TargetName="OverflowGrid" Property="DockPanel.Dock" Value="Bottom" />
```

There is a wrong Path.Data of the `MaterialDesignToolBarVerticalOverflowButtonStyle` in my opinion. I think that vertical toggle button must be rotated on 90 degrees.

### Version

4.6.0-ci220

**Comment by MichelMichels** [comment]
@p1eXu5 , can you provide a repository to showcase the problem you are facing? I can't figure out how to replicate your report.

**Comment by p1eXu5** [comment]
@MichelMichels, as I recall, I meant this - https://github.com/p1eXu5/MaterialDesignToolBarStyleProblem

**Comment by MichelMichels** [comment]
> @MichelMichels, as I recall, I meant this - https://github.com/p1eXu5/MaterialDesignToolBarStyleProblem

I'll look into it this weekend.

**Comment by nicolaihenriksen** [comment]
@p1eXu5 I believe this was solved by this PR #3000. Can you confirm that is the case?

**Comment by p1eXu5** [comment]
@nicolaihenriksen Yep, the problem has gone. Thanks!
